src/wp-includes/class-wp-block-bindings-source.php

```diff
@@ -75,14 +74,25 @@ public function __construct( string $name, array $source_properties ) {
 	 *
 	 * @since 6.5.0
```

Comment:
This is what I meant here:
```suggestion
	 * @since 6.5.0
	 * @since 6.7.0 `block_bindings_source_value` filter was added.
```